sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlUtils.java

```diff
@@ -54,27 +54,25 @@
   private ZetaSqlUtils() {}
 
   // Type conversion: ZetaSQL => Calcite
-  public static SqlTypeName zetaSqlTypeToCalciteTypeName(TypeKind zetaSqlType) {
-    switch (zetaSqlType) {
+  public static SqlTypeName zetaSqlTypeToCalciteTypeName(Type type) {
+    switch (type.getKind()) {
       case TYPE_INT64:
         return SqlTypeName.BIGINT;
-      case TYPE_NUMERIC:
```

Comment:
This is deleted, which doesn't match your no functional change description. What is going on here?

Reply:
This is actually dead code, we have never got NUMERIC working.